### ISSUE TITLE
Symbol not found _IOHIDManagerCreate

### ISSUE THREAD

**Issue by masterakos** [node-hid/node-hid#94]
Cloned the project and used ```npm install``` to execute all installation steps. 
```npm install``` output:
```
> node-hid@0.3.2 preinstall /Users/Tsalik/Projects/hidTest/node_modules/node-hid
> sh get-hidapi.sh

Cloning into 'hidapi'...
remote: Counting objects: 1965, done.
remote: Total 1965 (delta 0), reused 0 (delta 0), pack-reused 1965
Receiving objects: 100% (1965/1965), 2.79 MiB | 155.00 KiB/s, done.
Resolving deltas: 100% (1082/1082), done.
Checking connectivity... done.
\
> node-hid@0.3.2 install /Users/Tsalik/Projects/hidTest/node_modules/node-hid
> sh install.sh

  CC(target) Release/obj.target/hidapi/hidapi/mac/hid.o
../hidapi/mac/hid.c:255:20: warning: comparison of integers of different signs: 'CFIndex' (aka 'long') and 'size_t' (aka 'unsigned long') [-Wsign-compare]
                if (chars_copied == len)
                    ~~~~~~~~~~~~ ^  ~~~
../hidapi/mac/hid.c:295:20: warning: comparison of integers of different signs: 'CFIndex' (aka 'long') and 'size_t' (aka 'unsigned long') [-Wsign-compare]
                if (used_buf_len == len)
                    ~~~~~~~~~~~~ ^  ~~~
2 warnings generated.
  LIBTOOL-STATIC Release/hidapi.a
  CXX(target) Release/obj.target/HID/src/HID.o
  SOLINK_MODULE(target) Release/HID.node
  SOLINK_MODULE(target) Release/HID.node: Finished
0.12.2
nan@1.7.0 node_modules/nan
```
So far so good I suppose..
Then tried simply to ```node src/show-devices.js```
Got the following:
```
dyld: lazy symbol binding failed: Symbol not found: _IOHIDManagerCreate
  Referenced from: /Users/Tsalik/Projects/hidTest/node_modules/node-hid/build/Release/HID.node
  Expected in: dynamic lookup

dyld: Symbol not found: _IOHIDManagerCreate
  Referenced from: /Users/Tsalik/Projects/hidTest/node_modules/node-hid/build/Release/HID.node
  Expected in: dynamic lookup

Trace/BPT trap: 5
```

Also to mention that trying to install node-hid using npm repository also fails:
https://gist.github.com/masterakos/ea459e1b2a131c38af70#file-node-hid-npm-install-fails
And the debug log:
https://gist.github.com/masterakos/bbe451df52276b99a78e#file-node-hid-npm-install-debug-log

**Comment by ril3y** [comment]
Bump on this.  I have the exact same issue.

**Comment by holocron** [comment]
same issue under mac os x both 10.9 and 10.10. looks like IOKit lookup problem

**Comment by masterakos** [comment]
Indeed, problem solved by editing binding.gyp before ```npm install```. Thank you @holocron (Y)

**Comment by natevw** [comment]
I included https://github.com/masterakos/node-hid/commit/fea83535e1d65bfd296f7c9558626ab19d29c53a in https://github.com/node-hid/node-hid/pull/95, works a treat!

**Comment by natevw** [comment]
This should be fixed via #95 as of node-hid@0.4.0.
